### PR TITLE
Comments out Adrenals

### PR DESCRIPTION
**Pull request by Sarcalogo**
As it was in the old Hippie code.

### CODE REVIEW

**Comment by tortellinitony** [comment]
comment out meth and bath salts recipes too

**Comment by Sarcalogo** [comment]
One step at a time.

**Comment by Anderbergst** [comment]
comment out changeling adrenals too

**Comment by Sarcalogo** [comment]
One step. At a time.

File: code/game/objects/items/weapons/implants/implant_misc.dm [hunk 1]
```diff
@@ -14,12 +14,13 @@
...
-/obj/item/weapon/implant/adrenalin
+/*/obj/item/weapon/implant/adrenalin
 	name = "adrenal implant"
 	desc = "Removes all stuns and knockdowns."
 	icon_state = "adrenal"
 	origin_tech = "materials=2;biotech=4;combat=3;syndicate=4"
 	uses = 3
+	*/
 
 /obj/item/weapon/implant/adrenalin/get_data()
```
Comment (ZachDeava) [inline code/game/objects/items/weapons/implants/implant_misc.dm]: If you're commenting out the object, comment out all its procs too.

**Comment by Sarcalogo** [comment]
Going to have to make a Map Edit to remove the adrenalin implant object from Z2, can't do that now, going to sleep. Wait a few hours, maybe 12 or 16.

**Comment by BTFO** [comment]
Please don't comment out any code, just remove it. If we ever need it again we can just go back in time

**Comment by TheOneAndOnlyCreeperJoe** [comment]
There is literally no benefit to removing code over commenting it out. Emphasis on code.

**Comment by BTFO** [comment]
@TheOneAndOnlyCreeperJoe There is, it's called cleaning up your shit. There is no real need to keep commented out code inside the codebase as you have full git history so getting it back is easy.

Asides from that, why comment it out or remove it? Just prevent users from getting it whilst allowing admins to spawn it if they please

**Comment by JohnGinnane** [comment]
original PR in question: https://github.com/HippieStationCode/HippieStation13/pull/3198

This also have them commented out in there

**Comment by Sarcalogo** [comment]
Okay it's done.

**Comment by KittyKathy** [comment]
Nah don't merge this as it is only removing one form of antistuns therefore creating a bigger imbalance.  If you are not going to do this right and remove them all at once just close this now and I will do it myself.

**Comment by KittyKathy** [comment]
Also yeah don't comment it out just outright delete the code because it is not hard to get it back and it servers no purpose being in comments in the code. Plus we will never being back antistuns when they are gone so no reason to have it commented out in the code it just takes up space. Commenting out instead of removing is really only necessary when the change may be reverted, this is not the case here as once it is gone it's gone for good.

**Comment by Sarcalogo** [comment]
Almost all of the other anti-stuns (chemicals) were commented out already look at the code.

**Comment by KittyKathy** [comment]
I don't see any PR for that here no antistuns were removed at all. I am looking at this file https://github.com/HippieStation/HippieStation13/blob/master/code/modules/reagents/chemistry/reagents/drug_reagents.dm
and this file
https://github.com/HippieStation/HippieStation13/blob/master/code/modules/reagents/chemistry/reagents/medicine_reagents.dm
??????

**Comment by Sarcalogo** [comment]
What the hell? Why does my file have that shit commented out?

**Comment by Sarcalogo** [comment]
And yours doesn't?

**Comment by JohnGinnane** [comment]
Keep in mind that the old codebase is:
https://github.com/HippieStationCode/HippieStation13/
While the new one is:
https://github.com/HippieStation/HippieStation13/

**Comment by Sarcalogo** [comment]
No, Kitty Kathy I implore you to look again. Search this:

/* Get ye gone, anti stuns. */

**Comment by KittyKathy** [comment]
Click on the links you idiot and look through the files, you are looking at the old pre rebase code.

**Comment by Sarcalogo** [comment]
That's impossible, I downloaded and cloned this repository and that's what the files say.

**Comment by Sarcalogo** [comment]
Well whatever, that just makes things easier.

**Comment by Sarcalogo** [comment]
https://github.com/HippieStation/HippieStation13/pull/53

**Comment by KittyKathy** [comment]
Actually now I see what is going on...what the hell happened here.
If you go into the file https://github.com/HippieStation/HippieStation13/tree/master/code/modules/reagents
you will see a folder called chemistry which has all the new rebase stuff and the readded old stuff we have but in that same folder we have the old files from pre rebase.
So we basically have two files that have the medicine code one pre rebase and one with the new stuff.
Pre rebase medicine file
https://github.com/HippieStation/HippieStation13/blob/master/code/modules/reagents/Chemistry-Reagents/Medicine-Reagents.dm

Rebase medicine file
https://github.com/HippieStation/HippieStation13/blob/master/code/modules/reagents/chemistry/reagents/medicine_reagents.dm

So can anyone explain what the hell is going on? which file are we actually using ingame now to decide what medicine does. Why is it like this and someone fix this.

**Comment by Pyko1** [comment]
wew wew wew Ctrl+C +Ctrl+V mastercoding!

**Comment by Sarcalogo** [comment]
I figured it out, don't worry about it. This pull request was finished ages ago, but now I have to fix my 2nd one, should take 5 minutes.

**Comment by ZachDeava** [comment]
So, are you going to be updating this PR to address all the feedback or making a new one?

**Comment by ZachDeava** [comment]
@KittyKathy I'd appreciate it if you kept it more civil and professional on the GitHub.

**Comment by Sarcalogo** [comment]
This pull request is complete, the feedback mainly focused on removing OTHER anti-stuns (they were all removed on the original code base) but I don't want to lump everything together, because that means I have to do everything at once. There are a lot of ant-stuns in the game and I just removed 80% of them between these two pull requests. The hard part comes from identifying the rest so people don't abuse it.

**Comment by KittyKathy** [comment]
I don't have to be civil with people I don't like especially ones who don't listen to what I said all he had to do was click the link to see what I was talking about. also he is still commenting it out instead of deleting the code and has not given a reason why it is better to comment it out instead of removing it.

Also the main issue with this is it only removes some antistuns and leaves some in creating a imbalance in the game it should be all or nothing just like the orginal PR to remove antistuns pre rebase.

**Comment by Sarcalogo** [comment]
Wow, you don't like me, that's sad. Yes I clicked the link, but you were confused too due to the duplicate files, so don't act as if I did something wrong. It's the standard procedure here to comment things out instead of deleting them, that's what I've been told and I won't change it because you told me to. 

I've also removed now, 90% of anti stuns that are used by players between this pull request and the 2nd updated pull request. There isn't much left, and as soon as I am not busy I can do those. What is the big deal?

**Comment by Sarcalogo** [comment]
Or why not just do the rest yourself it you think it's so important?

**Comment by KittyKathy** [comment]
Link me these "stand procedures" also you were not supposed to just click it you were supposed to read the file. I'm the one confused? you were the one looking at the wrong file and saying things that were plain false. I'll say it again you have no idea what you are doing and if I was not here you would be screwing everything up so we would be worse off than before you did anything.

You were about to remove adrenals leaving in all the chemical stimulants like meth I said I would do It would remove it but you don't listen at all. I'm just going to stop responding to your PRs and do my own as it is a waste of time helping you and it would be better for everyone to see you screw up everything so we can have a new meme dong code 2.0 right here.

**Comment by KittyKathy** [comment]
If this gets merged I'm not going to even bother with trying to fix your mess someone else can do it, I hope it's the guy who merges it.

**Comment by Sarcalogo** [comment]
>Link me these "stand procedures

That is what JohnFG or some other Coder told me when I just outright removed the reactive teleport armor, they told me to comment it out. Or maybe it was something else, but they told me.

>You were supposed to read the file.

I did read it. I then said I was confused as to why it was different from a file that I knew  I downloaded from here. As far as I knew that would have been impossible unless someone had re-reverted the supposed commenting out of the chemicals. It turns out that there were duplicate files, one with the chemicals commented out and ones without them being commented out. I was in the wrong folder, code/module/Chemistry Reagents not code/module/chemistry/reagents. It's an easy mistake to make, how can you blame me for that?

>I'll say it again you have no idea what you are doing and if I was not here you would be screwing everything up so we would be worse off than before you did anything.

I know exactly what am I doing, how do I not?

>You were about to remove adrenals leaving in all the chemical stimulants like meth

No, I was going to make another pull to do that separately, presuming they'd all be merged at the same time frame.

>I said I would do It would remove it but you don't listen at all.
But you didn't do it, so I did it instead. Now if you want you have less work to do if you want to remove the other things that need removing.

>I'm just going to stop responding to your PRs and do my own as it is a waste of time helping you
Why?

>If this gets merged I'm not going to even bother with trying to fix your mess
What mess?

**Comment by Sarcalogo** [comment]
Forget it Kitty Kathy, if you are so desperate to do it, then do it yourself, you're only delaying the removal of something that needs to be removed, but you just have to have things your way.

**Pull request closed without merging**